packages/adapters/examples/segmentationVolume/utils.ts

```diff
@@ -11,218 +11,213 @@ const { downloadDICOMData } = cornerstoneAdapters.helpers;
-    const { segmentationId, viewportIds } = state;
-    const segmentationIds = getSegmentationIds();
-    if (!segmentationIds.length) {
-        return;
-    }
+  const { segmentationId, viewportIds } = state;
+  const segmentationIds = getSegmentationIds();
+  if (!segmentationIds.length) {
+    return;
+  }
 
-    const segmentation =
-        csToolsSegmentation.state.getSegmentation(segmentationId);
+  const segmentation =
+    csToolsSegmentation.state.getSegmentation(segmentationId);
 
-    const { imageIds } = segmentation.representationData.Labelmap;
-
-    const segImages = imageIds.map(imageId => cache.getImage(imageId));
-    const referencedImages = segImages.map(image =>
-        cache.getImage(image.referencedImageId)
-    );
+  const { imageIds } = segmentation.representationData.Labelmap;
 
-    const labelmaps2D = [];
+  const segImages = imageIds.map((imageId) => cache.getImage(imageId));
+  const referencedImages = segImages.map((image) =>
+    cache.getImage(image.referencedImageId)
+  );
 
-    let z = 0;
+  const labelmaps2D = [];
 
-    for (const segImage of segImages) {
-        const segmentsOnLabelmap = new Set();
-        const pixelData = segImage.getPixelData();
-        const { rows, columns } = segImage;
+  let z = 0;
 
-        for (let i = 0; i < pixelData.length; i++) {
-            const segment = pixelData[i];
-            if (segment !== 0) {
-                segmentsOnLabelmap.add(segment);
-            }
-        }
+  for (const segImage of segImages) {
+    const segmentsOnLabelmap = new Set();
+    const pixelData = segImage.getPixelData();
+    const { rows, columns } = segImage;
 
-        labelmaps2D[z++] = {
-            segmentsOnLabelmap: Array.from(segmentsOnLabelmap),
-            pixelData,
-            rows,
-            columns
-        };
+    for (let i = 0; i < pixelData.length; i++) {
+      const segment = pixelData[i];
+      if (segment !== 0) {
+        segmentsOnLabelmap.add(segment);
+      }
     }
 
-    const allSegmentsOnLabelmap = labelmaps2D.map(
-        labelmap => labelmap.segmentsOnLabelmap
+    labelmaps2D[z++] = {
+      segmentsOnLabelmap: Array.from(segmentsOnLabelmap),
+      pixelData,
+      rows,
+      columns,
+    };
+  }
+
+  const allSegmentsOnLabelmap = labelmaps2D.map(
+    (labelmap) => labelmap.segmentsOnLabelmap
+  );
+
+  const labelmap3D = {
+    segmentsOnLabelmap: Array.from(new Set(allSegmentsOnLabelmap.flat())),
+    metadata: [],
+    labelmaps2D,
+  };
+
+  labelmap3D.segmentsOnLabelmap.forEach((segmentIndex) => {
+    const color = csToolsSegmentation.config.color.getSegmentIndexColor(
+      viewportIds[0],
+      segmentationId,
+      segmentIndex
     );
-
-    const labelmap3D = {
-        segmentsOnLabelmap: Array.from(new Set(allSegmentsOnLabelmap.flat())),
-        metadata: [],
-        labelmaps2D
+    const RecommendedDisplayCIELabValue = dcmjs.data.Colors.rgb2DICOMLAB(
+      color.slice(0, 3).map((value) => value / 255)
```

Comment:
This prevents a dciodvfy error caused by the alpha channel being included.  This is the only change here.